guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -288,7 +439,14 @@ Registry:: An archive of container images.
 However, hosts cannot push changes back to the registry.
 
 [[Repository]]
-Repository:: Provides storage for a collection of content.
+// TODO: reword
+Repository::
+A repository is a single source of content, usually in form of an URL.
+It is the smallest storage unit for software content in {Project}.
+A repository generally needs to be synchronized with a content source to obtain content, although you can also manually upload packages to {Project}.
+To be usable in {Project}, a remote repository must be of a _type_ supported by {Project}.
```

Comment:
What is a "remote" repository? Is this a Pulp term bleeding into the documentation here? I don't think it adds much meaning.

Reply:
Removed "remote" and reworded the complete entry.